src/actions/filterActions.js

```diff
@@ -5,15 +5,23 @@ import * as eventActions from 'actions/eventActions'
 import * as filterService from 'services/filterService'
 
 export const CHANGE_EVENT_FILTER = 'CHANGE_EVENT_FILTER'
+export const CLEAR_EVENT_FILTER_INCIDENTID = 'CLEAR_EVENT_FILTER_INCIDENTID'
 
-export const changeEventFilter = (history) => (filter) => {
-  filterService.getUrlFromFilter(history, filter)
+export const changeEventFilter = (history, urlLoader) => (filter) => {
```

Comment:
This seems to be breaking the build/tests because the second argument isn't passed everywhere this method is called; if you default it to filterService.getUrlFromFilter, that should fix it. Alternatively, you could change the tests to pass in the getUrlFromFilter (or other url creation function).